packages/runtime/src/runtimes/react/controls/rich-text-v2/EditableTextV2/editable-text-v2.tsx

```diff
@@ -77,7 +79,57 @@ export function EditableTextV2({ text, control }: Props) {
   }, [editor, control])
 
   const renderElement = useCallback(({ attributes, children, element }: RenderElementProps) => {
+    const blockStyles = [useStyle({ margin: 0 })]
+
     switch (element.type) {
+      case BlockType.Text:
+        return (
+          <span {...attributes} className={cx(...blockStyles)}>
```

Comment:
Should we be composing `attributes.className` here? I imagine once you make the text align control which will compose rendering that might become important?

Reply:
Currently  `attributes` doesn't contain a classname because there is no way to compose the rendering just yet. 

<img width="938" alt="CleanShot 2023-05-12 at 16 39 08@2x" src="https://github.com/makeswift/makeswift/assets/20950876/d378a550-d5f0-4807-b73a-268243f9f2b9">

It might have it when I am done with textAlign though! TBH I can't totally visualize it yet.